bazel/foreign_cc/BUILD

```diff
@@ -187,6 +187,24 @@ cc_library(
     deps = ["unicode_icu_build"],
 )
 
+configure_make(
```

Comment:
Please move to contrib/isa_l/compression/source/common.

Reply:
OK, I will do that next. Thanks!